ui/app/common/components/pnc-admin-panel/pncAdminPanel.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2019 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+(function () {
+  'use strict';
+
+  /**
+   * The component for Admin panel for generic settings
+   */
+  angular.module('pnc.common.components').component('pncAdminPanel', {
+    bindings: {
+    },
+    templateUrl: 'common/components/pnc-admin-panel/pnc-admin-panel.html',
+    transclude: true,
```

Comment:
Does the admin panel need to transclude any content? If not then it's better to remove this!